scripts/lib/upgrade-zulip-stage-2

```diff
@@ -35,6 +35,17 @@ logging.Formatter.converter = time.gmtime
 logging.basicConfig(format="%(asctime)s upgrade-zulip-stage-2: %(message)s",
                     level=logging.INFO)
 
```

Comment:
I think we need `install` to write `postgres_version` as well; maybe in its own commit?  I imagine we just want to write 12, the latest release that we have postgres configuration for.